pants-plugins/tests/python/internal_backend_test/sitegen/test_sitegen.py

```diff
@@ -196,11 +197,10 @@ def test_site_toc(self):
                                    self.precomputed,
                                    """
                                    {{#site_toc}}
-                                   DEPTH={{depth}} LINK={{link}} TEXT={{text}}
+                                   DEPTH={{depth}} LINK={{links}} HEADING={{heading}}
                                    {{/site_toc}}
                                    """)
-    self.assertIn(u'DEPTH=1 LINK=subdir/page1.html TEXT=東京 is Tokyo', rendered)
-    self.assertIn('DEPTH=1 LINK=subdir/page2.html TEXT=Page 2: Electric Boogaloo', rendered)
+    self.assertIn(u'DEPTH=1 LINK=None HEADING=test', rendered)
```

Comment:
I'm not sure I understand why the assertions that use to be here were removed. I think they are still valuable because they were exercising the behavior around how links to pages are rendered. Maybe you could render both the heading and the text, and assert with `DEPTH=... LINK=... HEADING=... TEXT=...`?

Come to think of it, it would be good to have a collapsible_heading example here as well.